internal/analyzers/skippers/skippers.go

```diff
@@ -39,6 +39,14 @@ func NewSkipper(ctx context.Context) Skipper {
 			"enterprise": func(_ collectors.CollectedData) bool {
 				return !context_utils.GetIsCloud(ctx)
 			},
+			"is_branch_protection_supported": func(data collectors.CollectedData) bool {
```

Comment:
Please combine this logic and "has_branch_protection_permissions" to a single skipper. and update the old policies to use it.

Reply:
But there may be policies that we don't have permissions but BP is supported, so we need the other skipper for them.